doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -293,36 +296,53 @@ Additional scenario XML elements:
   must exactly match the module tag in the GRUB multiboot cmdline.
 
 ``bootargs`` (a child node of ``os_config``):
-  For internal use only and is not configurable. Specify the kernel boot arguments
-  in bootargs under the parent of board_private.
+  Specify the kernel boot arguments in bootargs under the parent of board_private.
```

Comment:
```suggestion
  Specify the kernel boot arguments in ``bootargs`` under the parent of ``board_private``.
```

Parent is `board_private` ou should it be `os_config`?

Reply:
updated.

Should be ``board_private``. There are several``bootargs`` in place. However, for the service os, the ``bootargs`` under ``os_config`` should is not configurable (configurable="0") and default value is "SOS_VM_BOOTARGS" which indicates it refers to the ``bootargs`` under ``board_private``.

Reply:
So is the definition given on line 298 correct?

Reply:
I edited a wrong one, just add `` as tag. Thanks.